public/katex.css

```diff
@@ -0,0 +1,1118 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: "KaTeX_AMS";
+  src: url(fonts/KaTeX_AMS-Regular.woff2) format("woff2"),
```

Comment:
We should probably add all of these fonts as well, no?

Reply:
These are available at `node_modules/katex/dist/fonts` I don't think that we need to add them twice.

Reply:
There is nothing that copies them automatically, similar to the CSS:
<img width="627" alt="Screenshot 2022-10-04 at 09 23 20" src="https://user-images.githubusercontent.com/158411/193759180-d506a507-d44b-4c80-ae08-15bbc53ba967.png">

Reply:
Ok did:
- add a postinstall script, where we copy the font files into the `public` directory
- add all the `KaTeX_*` fonts to `.gitignore` to avoid checking them in
- move back to a minified version of `katex.css` and add it to a new `.prettierignore` file